src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -119,8 +119,8 @@ def fuzz(self,
         if fnmatch.fnmatch(file, 'report*') and unique_crash not in visited:
           visited.add(unique_crash)
           log_lines = utils.read_data_from_file(
-              os.path.join(subdir, file), eval_data=False)
-          fuzz_logs += str(log_lines) + '\n'
+              os.path.join(subdir, file), eval_data=False).decode('utf-8')
```

Comment:
This is a bug, you can see an example log
gsutil cat gs://fuzz-logs.internal.clusterfuzz.com/syzkaller_syzkaller/syzkaller_kasan_android_device/2020-06-10/03:11:36:378571.log

Basically,
>>> a = b'123'
>>> str(a)
"b'123'"
>>> a.decode('utf-8')
'123'

Reply:
I forgot to tell utils.read_data_from_file gives bytes in my last review.